### ISSUE TITLE
[TEST] TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form: Create

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#348]
Draft of template issue for uploading interview recordings 
---
**name:** Research: Create upload form for interview recordings   
**about:** The process to upload interview recordings to Internship's private drive after the interviews are done
**title:** TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form: Create
**labels:** 'feature: recording transcription', 'feature: research', 'Participant Type: missing', 'Research 000: missing', 'size: 0.5pt', 'role: UI/UX research', 'milestone: missing'
**assignees:** 

---
### Dependencies
- [x] `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Roll Call and Session Table` has been created

### Overview
We need to create a new form to upload the recordings made for RP[RP001].

### Action Items
- [ ] UX lead asks admin to add the team member working on this issue to the 1Password internship-UX's vault
#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [x] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [x] Find relevant wiki page for RP[RP001]
    - [x] Copy the link for the RP[RP001] wiki page.
    - [x] **Update Resource # 2.01 with the link you just copied. Place it into parentheses at the end of the line (move to issue creator actions)**
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- Customize Resource for this Research Plan's Google Drive Folder
    - [x] Open the Google Drive's Research by Type Folder (Resources  # 1.02)
    - [x] Open the `[Intern]` Folder
    - [x] Copy the link for the `RP[RP001]` folder
        1. Right-click on the `RP[RP001]` folder
        2. Choose "Get Link"
        3. Choose "Copy link"
    - [x] Update Resource # 2.02 with the link you just copied. Paste into parentheses at the end of the line.
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- [x] Open the `RP[RP001]` folder (Resources # 2.02) if not already open
    - [x] Copy the link to the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Roll Call and Session Table` file
        1. Right-click on the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Roll Call and Session Table` folder
        2. Choose "Get Link"
        3. Choose "Copy link"
    - [x] Update Resources # 2.03 by pasting the link you just copied inside parentheses at the end of the line 
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked  
####  Set up a folder for the recording upload form
- [x] Open the internship@hackforla.org Google Drive in a separate Chrome/browser profile window (you do not want to cross-contaminate this process with your personal browser profile)
   - If you are not sure how to do this, use the following steps to log in:
      1. Log into 1password (Resources # 1.03) 
           - Refer to the guide linked in Resources # 1.03.01 if you are unsure about any of the following steps
      2. Choose internship-UI/UX vault
      4. Choose <img src="https://user-images.githubusercontent.com/37763229/202265799-5ec6b622-1b77-44cb-95fd-1f89eb05a195.png" width=150px> to see login info for internship@hackforla.org account
      5. In Chrome Browser, open a new browser window using the 
      6. **ADD INSTRUCTIONS FOR HOW TO SETUP A PROFILE FOR THE INTERNSHIP ACCOUNT**
      7. Go to "drive.google.com"
      8. Sign in with the internship@hackforla.org account using the login information in 1Password
- [ ] Choose `TWE Interview Recordings by Plan #` folder in "My Drive"
- [ ] Create a new folder for the uploads in the `TWE Interview Recordings by Plan #` folder by choosing "New" and "New folder" on the left side bar
    <img width="127" alt="Screenshot 2023-01-27 at 2 34 43 PM" src="https://user-images.githubusercontent.com/57029070/215219196-2e8b1712-6c74-4893-a1aa-7db51ca48e19.png">
    <img width="200" alt="Screenshot 2023-01-27 at 2 35 00 PM" src="https://user-images.githubusercontent.com/57029070/215219378-052eefb3-64e5-447f-941d-1c2d04f3f1f2.png">       
- [ ] Name the "New folder" by copying and pasting: 
```
TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings
```
- [ ] Copy the link to the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings` folder 
    1. Right-click on the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings` folder
    2. Choose "Get Link"
    3. Choose "Copy link"
- [ ] Update Resources # 2.04 by pasting the copied link inside parentheses at the end of the line
- [ ] Choose `Update comment` in Github and make sure all the checkboxes above have been checked
####  Copy the Google Form Template
- [ ] Open the Google Form Template link (Resources # 1.05) in the internship@hackforla.org browser window
    1. Copy link to Resources # 1.05
    2. Open a new browser tab in the internship@hackforla.org browser window
    3. paste the link into the address bar
- [ ] Choose the `Use Template` button
- [ ] A message that says `Missing File Upload folders` pops up, select `Cancel` at this step ![Cancel](https://user-images.githubusercontent.com/57029070/211449280-b423d288-42f4-4c50-914c-06430524162f.png)
- [ ] Move the form template into the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings` folder by completing the following steps
    1. Choose the `Move to folder` icon ![Screenshot 2023-01-09 at 6 34 19 PM](https://user-images.githubusercontent.com/57029070/211449094-c5479c29-0351-486c-837f-d9cca4321aba.png)
    4. Choose `TWE Interview Recordings by Plan #`
    5. Choose `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings` folder
    6. Choose `Move Here`
    7. Choose `Dismiss` on the alert notification
- Rename the form by following the steps:
    - [ ] Copy
    ```
    TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form
    ``` 
    - [ ] Paste the text you copied into the top field of the file 
    <img width="600" alt="Screenshot 2023-01-27 at 2 52 03 PM" src="https://user-images.githubusercontent.com/57029070/215221211-c2f7fc32-ddaa-4ed8-b094-33623162af09.png">
- [ ] Refresh the browser tab with the Google form
- [ ] A message that says `Missing File Upload folders` pops up again, choose `Restore` at this step ![Restore](https://user-images.githubusercontent.com/57029070/211466563-f7bf3944-5db5-4e99-8729-05938af56244.png) (This will create a destination folder for the file uploads)
- A `Folders restored successfully` notification should flash at the bottom of the window
#### Customize the Recording Upload Google Form just made

- In the body text of "Finishing up: Sending your recording to TWE team" section, complete the following edits:
    - [ ] Copy **(ADD INSTRUCTIONS TO CUSTOMIZE PARTICIPANT TYPE CODE WHEN CUSTOMIZING THE ISSUE)**
    ```
    RP[RP001]-U000-[Replace PARTICIPANT CODE]000
    ``` 
    - [ ] Paste the copied text in the section after "Rename the recording file in the following format" (see the grayed out texts in the screenshot below)  <img width="600" alt="Screenshot 2023-01-17 at 4 46 59 PM" src="https://user-images.githubusercontent.com/57029070/213053453-d8a93e94-b0f8-444c-95ea-0c4e423d88de.png">
    - [ ] Make sure the text you just copied are in bold and italic in the Google Form
    - [ ] Single click on the link that says `[Replace TEMPLATE: Interview Roll Call and Session Table]`, and choose on the pencil icon to "Edit Link".   <img src="https://user-images.githubusercontent.com/57029070/211993183-50bd3f76-5298-4241-8f5a-91972e0a59d3.png" width="400" height="80">
    - [ ] Copy Resource # 2.04 (both the text and URL)
    - [ ] **In the "Link to" section, paste the link you just copied (NEED TO UPDATE INSTRUCTIONS)**
    - [ ] In the "Text to display" section, replace `[Replace TEMPLATE: Interview Roll Call and Session Table]` with 
    ```
    TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Roll Call and Session Table
    ```
    <img src="https://user-images.githubusercontent.com/57029070/211993361-ad9cd48f-52ba-4c1d-9e22-07e05a6c8e32.png" width="300" height="150">
    - [ ] Choose "OK"
- In the form edit, go to Settings
    - [ ] Make sure to toggle off the `Restrict to users in Hack for LA and its trusted organizations` option <img src="https://user-images.githubusercontent.com/57029070/211475661-ebad4f04-d6fa-48e8-a325-59346a163ee0.png" width="700" height="50">
    - [ ] Choose `Send` button (upper right of window)
    - [ ] Choose link <img src="https://user-images.githubusercontent.com/57029070/211476642-a4d010c8-a1ef-475f-809c-3e73095272ed.png" width="60" height="30">
    - [ ] Check the `Collect emails` box, and check the `Shorten URL` box, then choose `Copy`
    - [ ] Update Resource # 2.05 with the link you just copied. Place it into parentheses at the end of the line. 
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
#### Edit Response Destination
- In the form edit, choose `Responses`
- [ ] Choose 3 vertical dots   <img width="30" alt="Screenshot 2023-01-31 at 5 48 33 PM" src="https://user-images.githubusercontent.com/57029070/215925141-2a3f8145-08cc-4ba9-bbe2-507aed50e8b0.png">
- [ ] Choose `Select destination for responses`
- [ ] Choose `Create a new spreadsheet`
- [ ] Choose `Create`
- [ ] In the internship@hackforla.org browser, open the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings` folder (Resources # 2.04)(You may already have this open)
- [ ] Copy click on the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form (Responses)` spreadsheet
    1.  Right-click on the `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form (Responses)` spreadsheet
    2. Choose "Get Link"
    3. Choose "Copy link"
- [ ] Update Resources # 2.06 by pasting the link inside parentheses at the end of the line
- [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked off
- [ ] Close the internship@hackforla.org browser
#### Update Research Plan Wiki Page 
- [ ] Open `Wiki: Research Plan 014 ` (Resource 2.01)
- [ ] Choose "Edit" to edit the page
- Find the "Interview Recordings & Transcripts" section
- [ ] Add a new line and insert a "-" dash
- [ ] Copy and paste Resource # 2.05 (both the text and URL) at the end of this new line
- [ ] Choose "Save Page"
- [ ] Open the Google drive folder RP[RP001] (Resources # 2.02) 
- [ ] Check that `TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form` IS NOT saved in this folder
- [ ] Review with UX Lead
- [ ] Product sign-off

#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03 [1password](https://start.1password.com/signin?l=en)
    1.03.01 [Guide: Log into a Google Account with 2-Step Verification Using 1Password](https://docs.google.com/document/d/1hL3gYqoV6EWl4bUa2VYLgq0N__CjTvSw4Da2M4x2Hxk/edit) _If this is your first time logging into Internship's private drive, please follow the instructions to log in._
1.04 [TWE Interview Recordings by Plan #](https://drive.google.com/drive/folders/1ytcgb_j5Nsl6ZIhGUCeqQkKzgo2FohRS?usp=share_link) _This folder is only accessible using internship@hackforla.org's Google account_
1.05 [Interview Recording Upload Form Template](https://docs.google.com/forms/d/15rQs70ewlMcKGu5B2yrFQBKfenE3GUY76wLUrBGXEnc/template/preview) _This template is only accessible using internship@hackforla.org's Google account_

#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan [RP001]] **(https://github.com/hackforla/internship/wiki/Research-Output-Overview)** 
2.02 [Google Drive Folder: RP[RP001]] (https://drive.google.com/drive/folders/1uKUY_izpds3dHInNkiexZsU32ELF_Hze)
2.03 [TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Roll Call and Session Table] (https://docs.google.com/spreadsheets/d/1j7DJltYPago3pZy_M-H_H_jBwOrBu6AN2LavTWTy2JU/edit?usp=share_link)
2.04 [TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recordings]
2.05 [TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form]
2.06 [TWE: IS[21]: RP[RP001]: [Intern] [Exit] Interview Recording Upload Form (Responses)]

**Comment by pandanista** [comment]
@doctorsandy Would you please test out this template? Any feedback is appreciated. Feel free to use any research plan that use interview as the research method for testing. You can find all the research plans on [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview), or in this [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link). Let me know if you have any questions.

**Comment by pandanista** [comment]
Previous tester ran into issues with 1Password and Google account. Need to revise that section.

**Comment by doctorsandy** [comment]
Here is where I am stuck. I can't find where to "ADD INSTRUCTIONS FOR HOW TO SETUP A PROFILE FOR THE INTERNSHIP ACCOUNT". I am also unclear as to how to signin with the hackforla gmail address using 1password.
<img width="416" alt="Screenshot 2023-03-29 at 3 24 12 PM" src="https://user-images.githubusercontent.com/110558153/228647049-58450606-bc8b-43e3-800e-7498ec55e28c.png">

**Comment by doctorsandy** [comment]
Update: I managed to log into the Google drive with some trial and error on my own. My suggestion then is to re-write these steps for clarity.

Inclusive to these revisions, I suggest that it is explicitly stated to use "other signin method" and identify that one of the backup codes in the 1password vault should be used. I simply guessed that this strategy might be successful.

Once I was in the drive, I could not locate this folder:

<img width="403" alt="Screenshot 2023-03-29 at 3 48 30 PM" src="https://user-images.githubusercontent.com/110558153/228651614-8de58094-b1fa-4b10-b0c4-d42af48ddf68.png">
